src/pycram/yarp_utils/yarp_networking.py

```diff
@@ -0,0 +1,36 @@
+
+import yarp
+
+
+ACK_VOCAB = yarp.createVocab32('a','c','k')
+NO_ACK_VOCAB = yarp.createVocab32('n','a','c','k')
+
+
+def init_yarp_network():
```

Comment:
Also here, please add doc strings and type hints

Reply:
done